packages/experimental/Icon/src/Icon.tsx

```diff
@@ -16,7 +16,15 @@ function renderRasterImage(iconProps: IconProps) {
     rasterImageStyleCache({ width: width, height: height, tintColor: color }, [width, height, color])[0],
   );
 
-  return <Image source={iconProps.rasterImageSource.src} style={style} />;
+  return (
+    <Image
+      source={iconProps.rasterImageSource.src}
+      style={style}
+      accessible={true}
+      accessibilityRole="image"
+      accessibilityLabel={iconProps.accessibilityLabel}
```

Comment:
Post checkin comment: To avoid this in the future, we could do something where we pass in `...props` so that the user can specify any prop. I think that's a pattern we use elsewhere to avoid similar bugs.